caddyhttp/fastcgi/setup.go

```diff
@@ -53,15 +53,13 @@ func fastcgiParse(c *caddy.Controller) ([]Rule, error) {
 	var rules []Rule
 
 	for c.Next() {
-		var rule Rule
-
 		args := c.RemainingArgs()
 
 		if len(args) < 2 || len(args) > 3 {
 			return rules, c.ArgErr()
 		}
 
-		rule.Path = args[0]
+		rule := Rule{Path: args[0], ReadTimeout: 60 * time.Second}
```

Comment:
Thanks for pointing it out, @wendigo.

@mholt some configurations may rely on a read timeout of greater than 60 seconds, whether users know it or not. Affected users would have to update their Caddyfile with the new read_timeout property in order to preserve the previous behavior. And even if the property is used, the exact duration might not be known.

Some options:
1. I can remove the default value and users must explicitly declare the read_timeout
2. Make the default duration greater than the currently proposed one

Are there others?